api/envoy/config/overload/v3/overload.proto

```diff
@@ -93,6 +93,11 @@ message ScaleTimersOverloadActionConfig {
     // Adjusts the idle timer for downstream HTTP connections that takes effect when there are no active streams.
     // This affects the value of :ref:`RouteAction.idle_timeout <envoy_v3_api_field_config.route.v3.RouteAction.idle_timeout>`.
     HTTP_DOWNSTREAM_CONNECTION_IDLE = 1;
+
+    // Adjusts the timer for how long downstream clients have to finish transport-level negotiations before the connection is closed.
```

Comment:
nit: This line seem very long.  Can you rerun format?

Reply:
Fixed manually since the formatter doesn't seem to touch proto comments.